pyrefinebio/http.py

```diff
@@ -61,15 +70,23 @@ def put(url, payload=None):
 
 
 def get_by_endpoint(endpoint, params=None):
-    url = base_url + endpoint + "/"
+    config = Config()
```

Comment:
It seems like we're going to reread the config file every time someone makes any request. This seems like a good application of the Singleton Pattern. I think the easiest way to implement it in python that I can think of would be to add something like this to config.py:

```python
_config = None

def get_config():
  if not _config:
    _config = Config()

  return _config
```

And then just call `get_config()` everywhere you need it.

Reply:
Whoops, nvm, I see how you're already doing that in Config! Nice work!